recipe/script.js

```diff
@@ -0,0 +1,42 @@
+function validTextColor(strColor) {
+    const instanceStyle = new Option().style;
+    instanceStyle.color = strColor; 
+    return instanceStyle === strColor;
+}
+
+let backgroundColor = function() {
+    let userColor = prompt("What background color do you want?", 'rgb(164, 247, 222)');
+    if(!validTextColor(userColor)){
+      console.log("This is not a valid input");
+    }
+    return document.body.style.backgroundColor = userColor;
+  }
+  
+  backgroundColor();
```

Comment:
a recommendation for readability is to keep function and variable declarations near the top of your file and then method calls below that.

an example file would look like the following: 
```js
const someVariable = 1;
cost someList = [1,2,3,4,5,6];
const someFunction = (inputList, inputVariable) => {
   return inputList.includes(inputVariable);
}

console.log(someFunction(someList, someVariable))
```